templates/ts-empty/.dockerignore

```diff
@@ -0,0 +1,13 @@
+# configurations
```

Comment:
`dist` folder should be here too

Reply:
@B4nan Should this be in all ts templates as well?

Reply:
Is it possible to put the dockerignore next to Dockerfile? It's kinda weird to have those two files in different folders.

Reply:
> Is it possible to put the dockerignore next to Dockerfile? It's kinda weird to have those two files in different folders.

Not yet, it's possible in Docker 23, but we're still on Docker 20 on the platform.